Paccy10/olympus-api
# Feature(types): create property type

**What does this PR do?**

- Adds create type functionality

**Description of Task to be completed?**

- An authenticated admin user can create a new property type

**How should this be manually tested?**

- Checkout to the branch `ft-create-type`
- Run the app with `flask run`
- Go to postman and send a `POST` request to /types with the required payload
- A new property type should be added to the database

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
